doc/release-notes/5.6-release-notes.md

```diff
@@ -0,0 +1,112 @@
+
+Dataset metadata can be retrieved, set, and updated using a new, flatter JSON-LD format - following the format of an OAI-ORE export (RDA-conformant Bags), allowing for easier transfer of metadata to/from other systems (i.e. without needing to know Dataverse's metadata block and field storage architecture). This new API also allows for the update of terms metadata (#5899).
+
+This development was supported by the [Research Data Alliance](https://rd-alliance.org), DANS, and Sciences PO and follows the recommendations from the [Research Data Repository Interoperability Working Group](http://dx.doi.org/10.15497/RDA00025).
+
+### Direct Upload API Now Available for Adding Multiple Files Metadata to the Dataset
+
+Users can now add metadata of multiple files to the dataset once the files exists in the s3 bucket using the direct upload API. For more information, see the [Direct DataFile Upload/Replace API section](https://guides.dataverse.org/en/5.6/developers/s3-direct-upload-api.html) of the Dataverse Software Guides.
+
+## Major Use Cases
+
+Newly-supported major use cases in this release include:
+
+- Users can create Private URLs that anonymize dataset metadata, allowing for double blind peer review. (Issue #1724, PR #7908)
+- Users can download Guestbook records using a new API. (Issue #7767, PR #7931)
+- Users can update terms metadata using the new semantic API. (Issue #5899, PR #7414)
+- Users can retrieve, set, and update metadata using a new, flatter JSON-LD format. (Issue #6497, PR #7414)
+- Administrators can use the Traces API to retreive information about specific types of user activity  (Issue #7952, PR #7953)
+
+## Notes for Dataverse Installation Administrators
+
+### New Database Constraint
+
+A new DB Constraint has been added in this release. Full instructions on how to identify whether or not your database needs any cleanup before the upgrade can be found in the [Dataverse software GitHub repository](https://github.com/IQSS/dataverse/blob/develop/scripts/issues/7451/PRE-RELEASE-INFO.txt). This information was also emailed out to Dataverse installation contacts.
+
+### Payara 5.2021.4 (or Higher) Required
+
+Some changes in this release require an upgrade to Payara 5.2021.4 or higher.
+
+Instructions on how to update can be found in the [Payara documentation](https://docs.payara.fish/community/docs/5.2021.4/documentation/user-guides/upgrade-payara.html) We've included the necessary steps below, but we recommend that you review the Payara upgrade instructions as it could be helpful during any troubleshooting.
+
+### Enhancement to DDI Metadata Exports
+
+To increase support for internationalization and to improve compliance with CESSDA requirements, DDI exports now have a holdings element with a URI attribute whose value is the URL form of the dataset PID.
+
+## New JVM Options and DB Settings
+
+:AnonymizedFieldTypeNames can be used to identify specific fields that will be anonymized when a user creates an anonymized Private URL.
```

Comment:
suggested change: :AnonymizedFieldTypeNames can be used to enable creation of anonymized Private URLs and to specify which fields will be anonymized.

Reply:
🥂 

https://github.com/IQSS/dataverse/pull/8022/commits/669adb3022c091b69811e8f903edb095cc1342c6